src/appshell/CMakeLists.txt

```diff
@@ -87,9 +87,20 @@ set(MODULE_SRC
     ${CMAKE_CURRENT_LIST_DIR}/view/preferences/iopreferencesmodel.h
     ${CMAKE_CURRENT_LIST_DIR}/view/preferences/commonaudioapiconfigurationmodel.cpp
     ${CMAKE_CURRENT_LIST_DIR}/view/preferences/commonaudioapiconfigurationmodel.h
+    ${CMAKE_CURRENT_LIST_DIR}/view/framelesswindow/framelesswindowmodel.cpp
+    ${CMAKE_CURRENT_LIST_DIR}/view/framelesswindow/framelesswindowmodel.h
+    ${CMAKE_CURRENT_LIST_DIR}/view/framelesswindow/abstractnativeeventfilter.cpp
+    ${CMAKE_CURRENT_LIST_DIR}/view/framelesswindow/abstractnativeeventfilter.h
     ${DOCKWINDOW_SRC}
     )
 
+if (OS_IS_WIN)
+    set(MODULE_SRC ${MODULE_SRC}
+        ${CMAKE_CURRENT_LIST_DIR}/view/framelesswindow/windowsnativeeventfilter.cpp
+        ${CMAKE_CURRENT_LIST_DIR}/view/framelesswindow/windowsnativeeventfilter.h
```

Comment:
I think it's better to move these files to `src/appshell/internal/platform/win`

Reply:
thanks! fixed